source/MRViewer/MRShortcutManager.h

```diff
@@ -52,7 +76,9 @@ class MRVIEWER_CLASS ShortcutManager
     MRVIEWER_API virtual bool processShortcut( const ShortcutKey& key, Reason = Reason::KeyDown ) const;
 
     // make string form key and returns it
-    MRVIEWER_API static std::string getKeyString( const ShortcutKey& key );
+    MRVIEWER_API static std::string getKeyString( const ShortcutKey& key, bool respectLastKey = true );
```

Comment:
better get tree separate functions
```c++
MRVIEWER_API static std::string getModifierString( int modifier );
MRVIEWER_API static std::string getKeyString( int key );
static std::string getKeyFullString( const ShortcutKey& key )
{
    return getModifierString( key.mod ) + getKeyString( key.key )
}
```
so you can use it in drawer without duplicating code from  ShortcutManager